firebase.json

```diff
@@ -20,16 +20,22 @@
   ],
   "emulators": {
     "functions": {
+      "host": "0.0.0.0",
```

Comment:
using `0.0.0.0` to allow all IPV4 requests to be captured. Otherwise `127.0.0.1` is used, which causes issues when trying to connect from external computer